.github/workflows/test.yaml

```diff
@@ -68,6 +68,21 @@ jobs:
       - name: Test kind works and there is no cluster started
         run: |
           [[ $(kind get clusters | wc -l) -eq 0 ]]
+          
+  test-with-macos-install-only:
+    runs-on: macos-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```

Comment:
```suggestion

  test-with-macos-install-only:
    runs-on: macos-latest
    steps:
      - name: Checkout
        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1

      - name: Setup docker
        run: |
          brew install docker
          colima start
```

Reply:
Tested this fix on my fork, it works: https://github.com/musse/kind-action/pull/1

Check this thread for details: https://github.com/actions/runner-images/issues/17#issuecomment-1445517870

Reply:
i don't think this is right; you shouldn't need `colima` running if the action is in `install_only` mode